scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/goals/RecurrenceGoal.java

```diff
@@ -72,6 +80,9 @@ protected RecurrenceGoal fill(RecurrenceGoal goal) {
         throw new IllegalArgumentException(
             "creating recurrence goal requires non-null \"every\" duration interval");
       }
+      if (every.min.isNegative() || every.max.isNegative()) {
+        throw new IllegalArgumentException("Duration passed to RecurrenceGoal as the goal's recurrence interval cannot be negative!");
```

Comment:
This would fail the entire run, I'm not sure if it should just fail the goal. I think that makes sense but it might be a good idea to fail entirely just so that no time is wasted running scheduling prior to the goals actually being properly specified. What are your thoughts?

Reply:
This is literally the conversation we're having here https://jpl.slack.com/archives/C0163E42UBF/p1656012405258839

Reply:
Oh neat, then this seems like its the correct behavior per [this message](https://jpl.slack.com/archives/C0163E42UBF/p1656016362871939?thread_ts=1656012405.258839&cid=C0163E42UBF).